config/initializers/devise.rb

```diff
@@ -402,7 +402,12 @@
   config.expire_after = expire_after.days
 
   if ENV['OKTA_DOMAIN'].present?
-    require_dependency 'omni_auth/strategies/custom_okta'
+    require 'omni_auth/strategies/custom_okta'
```

Comment:
Was this an intentional change from require_dependency to require?

Reply:
Yes. Middleware cannot be reloaded so _dependency is useless

Reply:
thanks